locals.tf

```
@@ -66,5 +66,21 @@ locals {
       name  = "pe.ruby.com.br"
       value = "frevo-on-rails.github.io"
     }
+
+    # Grupo de Usuários de Ruby de Goiás (go.ruby.com.br)
+    # go.ruby.com.br -> https://github.com/Guru-goias/gurugo
+    go = {
+      type  = "CNAME"
+      name  = "go.ruby.com.br"
+      value = "guru-goias.github.io/gurugo"
```

```suggestion
      value = "guru-goias.github.io"
```

Como é um registro CNAME, ele não suporta path. Nesse caso, você precisaria remover o `/gurugo` aqui e nas configurações do [GitHub Pages](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain), adicionar os domínios novos. Não sei se o GitHub suporta mais de um domínio ao mesmo tempo, mas vale a pena tentar.

Quando configurar lá e modificar aqui, me avisa para eu testar antes de fazer o merge. Valeu!

---

cc @alexcalaca

---

@wagner fiz aqui o ajuste